packages/@lwc/engine-server/src/renderer.ts

```diff
@@ -4,8 +4,14 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-import { isUndefined, isNull } from '@lwc/shared';
-import { Renderer } from '@lwc/engine-core';
+import {
+    isUndefined,
+    isNull,
+    isBooleanAttribute,
+    isGlobalHtmlAttribute,
+    isAriaAttribute,
+} from '@lwc/shared';
+import { Renderer, getAttrNameFromPropName } from '@lwc/engine-core';
```

Comment:
Would it make sense to move `getAttrNameFromPropName` to @lwc/shared? May be in a separate PR.